SWLOR.Game.Server/Perk/ForceSense/AnimalBond.cs

```diff
@@ -33,6 +51,136 @@ public float CooldownTime(NWCreature oPC, float baseCooldownTime, int spellTier)
 
         public void OnImpact(NWCreature creature, NWObject target, int perkLevel, int spellTier)
         {
+            ApplyEffect(creature, target, spellTier);
+        }
+
+        private void ApplyEffect(NWCreature creature, NWObject target, int spellTier)
+        {
+            Effect effect = _.EffectDominated();
+            int animalbondcreaturerating;
+            float duration = 300f;
+            int challengeratingxp = ((int)_.GetChallengeRating(target));
+
+            switch (spellTier)
```

Comment:
Please remove this if you aren't going to finish it up. Throw it in another branch and see if someone will pick it up. Code's a mess as it is, no need to make it worse :P